charts/allure/values.yaml

```diff
@@ -10,11 +10,13 @@ image:
   ui:
     repository: frankescobar/allure-docker-service-ui
     pullPolicy: Always
+    pullSecret: ""
```

Comment:
You are consuming previous this as `image.ui.pullSecrets` (in plural). You should change here or in the deployments

I also think the default value shouldn't be empty string, but `{}`